### PR TITLE
Remove gulp-load-plugins dependency

### PR DESCRIPTION
**Pull request by joemaller**
Simplify the gulpfile by removing gulp-load-plugins dependency. Some
whitespace cleanup and formatting to match normal NPM conventions.

### CODE REVIEW

**Comment by austinpray** [comment]
https://travis-ci.org/roots/sage/jobs/60721031#L264

File: gulpfile.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
-// ## Globals
```
Comment (austinpray) [inline gulpfile.js]: Add this back in

Comment (joemaller) [inline gulpfile.js]: ok, sorry about that, I see it's a comment not a directive...

Comment (austinpray) [inline gulpfile.js]: It makes everything look pretty when you run docco https://austinpray-web.s3.amazonaws.com/temp-roots/gulpfile.html

**Comment by JulienMelissas** [comment]
+1 for removing gulp-load-plugins dependency but I'm a fan of the space-alignment personally - seems much more readable.

**Comment by austinpray** [comment]
https://github.com/joemaller/sage/pull/1

**Comment by michaelbeil** [comment]
:dart:

**Comment by austinpray** [comment]
squash then looks good to go!

**Comment by michaelbeil** [comment]
:point_right:  :red_circle:

**Comment by joemaller** [comment]
squashed!

**Comment by austinpray** [comment]
tyvm!

**Comment by michaelbeil** [comment]
:tada:

**Pull request merged**